lib/trento_web/templates/session/new.html.eex

```diff
@@ -24,7 +24,7 @@
             Password
           </label>
           <div class="mt-1">
-            <input id="user_password" name="user[password]" type="password" autocomplete="current-password" required class="appearance-none block w-full px-3 py-2 border border-gray-300 rounded-md shadow-sm placeholder-gray-400 focus:outline-none focus:ring-jungle-green-500 focus:border-jungle-green-500 sm:text-sm">
+            <input id="user_password" name="password" type="password" autocomplete="current-password" required class="appearance-none block w-full px-3 py-2 border border-gray-300 rounded-md shadow-sm placeholder-gray-400 focus:outline-none focus:ring-jungle-green-500 focus:border-jungle-green-500 sm:text-sm">
```

Comment:
shouldn't we port this view completely to React SPA?

Reply:
Yes, this is a leftover, I will remove the file, good catch :D